docs/man/rpmkeys.8.scd

```diff
@@ -6,76 +6,128 @@ rpmkeys - RPM Keyring
+	keys.
 
-Digital signatures cannot be verified without a public key. An ASCII
-armored public key can be added to the *rpm* persistent keyring using
-*--import*.
+*--rebuild*
+	Recreate the public key storage. Update to the latest format and drop
+	unreadable keys.
 
-The following commands are available for manipulating the persistent
-rpm keyring:
+# ARGUMENTS
 
-*rpmkeys* {*-l*|*--list*} [_FINGERPRINT_...]
+_FINGERPRINT_
+	The handle used for all operations on the keys.
+_PACKAGE_FILE_
+	An *rpm* package file or a manifest.
+_PUBKEY_
+	An ASCII-armored OpenPGP public key (aka certificate).
 
-List currently imported public key(s) (aka certificates) by their
-fingerprint and user ID. If no fingerprints are specified, list all keys.
+# OPTIONS
 
-The fingerprint is the handle used for all operations on the keys.
+See *rpm.common*(8) for the options common to all *rpm* executables.
 
-*rpmkeys* {*-x*|*--export*} [_FINGERPRINT_ ...]
+# REBUILD OPTIONS
 
-Output the key(s) using an ASCII-armor encoding.
+*--from* <*fs*|*openpgp*|*rpmdb*>
 
-Exporting allows for inspecting the data with specialized tools, such
-as Sequoia or GnuPG. For example:
+Use the keys from the specified backend to rebuild the currently
+configured keystore backend.
+This can be used to convert from one key storage to another.
 
-*rpmkeys --export 771b18d3d7baa28734333c424344591e1964c5fc | sq inspect *
+# OUTPUT
+*--checksig*
+	Non-verbose output:
+```
+<_PACKAGE_FILE_>: <element> <element> <OK|NOT OK>
+```
 
-*rpmkeys* {*-e*|*--erase*|*-d*|*--delete*} _FINGERPRINT_ ...
+	Verbose output:
+```
+<_PACKAGE_FILE_>:
+    <element>: <OK/NOTFOUND/BAD>
+    ...
+```
 
-Erase the key(s) designated by _FINGERPRINT_. For example:
+*--list*
+```
+<fingerprint> <name> <userid> public key
+```
 
-*rpmkeys* *--erase 771b18d3d7baa28734333c424344591e1964c5fc*
+# CONFIGURATION
 
-Rebuild the key storage:
+There are several configurables affecting the behavior of this
+verification, see *rpm.config*(5) for details:
+- *%\_keyring*
+- *%\_keyring_path*
+- *%\_pkgverify_flags*
+- *%\_pkgverify_level*
 
-*rpmkeys* *--rebuild*
+# EXIT STATUS
+On success, 0 is returned, a non-zero failure code otherwise.
 
-Recreate the public key storage. Update to the latest format and drop
-unreadable keys.
+# EXAMPLES
 
-*--from* _BACKEND_
+*rpmkeys --export 771b18d3d7baa28734333c424344591e1964c5fc | sq inspect*
+	Export a key for inspecting with sequoia-sq.
 
-Use the keys from _BACKEND_ as content to build the configured
-backend.  Valid values for _BACKEND_ are *rpmdb*, *fs*, *openpgp*.
-This can be used to convert from one key storage to another.
+*rpmkeys --erase 771b18d3d7baa28734333c424344591e1964c5fc*
+	Erase key 771b18d3d7baa28734333c424344591e1964c5fc from the keyring
 
-# SEE ALSO
+*rpmkeys -K hello-2.0-1.x86_64.rpm*
+	Verify hello-2.0-1.x86_64.rpm package file.
 
-*popt*(3), *rpm*(8), *rpmdb*(8), *rpmsign*(1), *rpm2cpio*(1),
-*rpmbuild*(1), *rpmspec*(1)
+	```
+	$ rpmkeys -K hello-2.0-1.x86_64.rpm
```

Comment:
I'm thinking that maybe this could be instead:

```
Example output:
hello-2.0-1.x86_64.rpm: digests SIGNATURES NOT OK
```

WDYT? That would avoid the duplicated command as well as the extra nesting.

Reply:
It would be also more consistent with the OUTPUT section.

Reply:
Nah, "Example output" probably isn't right, this is already under EXAMPLES... :sweat_smile:  So perhaps just "Output:"?

Reply:
Detaching the example output from the command seems somehow wrong, because ... well, you kinda expect the example to be a whole.

Reply:
We could also just drop the example output. People will find out by trying :sweat_smile:

Reply:
True... :smile: And yeah, dropping is also an option of course, though it doesn't hurt to see what the output actually looks like without invoking the command... :smile:

Reply:
Ah, right, that's a good point. Might as well then drop the output from here, indeed. It's not super critical.

Reply:
Gone :sweat_smile: 

We can always add them later if we find a nice way to do it. For example the rpmspec examples I'd hate to lose.